values.yaml

```diff
@@ -37,6 +37,11 @@ global:
   # chart. See https://kubernetes.io/docs/concepts/policy/pod-security-policy/
   enablePodSecurityPolicies: false
 
+  # Gossip encryption key. To disable gossip encryption, set this to null.
+  # Note: DO NOT use the provided key in production. Generate your own with
+  # "consul keygen" and paste it here.
+  gossipKey: cg8StVXbQJ0gPvMd9o7yrg==
```

Comment:
It is generally not a good idea to include sensitive information in a configuration file since many people check it into source control. Like we do elsewhere in the chart, we should recommend folks create a Kubernetes secret that contains this information, and include `secretName` and `secretKey` fields here instead. The `.server.enterpriseLicense` or `syncCatalog.aclSyncToken` are good examples of this pattern.

Reply:
Of course you're right. Not sure why I didn't do it this way in the first place.